www/src/lore/Main_Page.md

```diff
@@ -77,8 +96,33 @@ Landfall](/wiki/Vision_of_Landfall "wikilink").
 -   [Fae](/wiki/Fae "wikilink"): Earlier human "settlers" of the islands, they
     are imbued with natural magic and come in blue (sea) and green
     (land) varieties. They are nomadic and live in small tribes.
+    
+</details>
+
+### How To Play
```

Comment:
we probably want a real tutorial

Reply:
that's in `/wiki/How_to_Play`, which I also linked in `Main_Page.md`!